lib/LANraragi/Utils/Archive.pm

```diff
@@ -247,10 +247,10 @@ sub get_filelist {
 
     @files = sort { &expand($a) cmp &expand($b) } @files;
 
-    # Move any pages containing "credit" to the end of the array.
-    my @credit_pages     = grep { /credit/i } @files;
-    my @non_credit_pages = grep { !/credit/i } @files;
-    @files = ( @non_credit_pages, @credit_pages );
+    # Move front cover pages to the start of a gallery, and miscellaneous pages such as translator credits to the end.
+    my @cover_pages      = grep { /^(?!.*(back|end|rear|recover)).*cover.*/i } @files;
+    my @credit_pages     = grep { /^999999|^bumper|^ramble\.[^\.]*$|^end_card_save_file|notes\.[^\.]*$|note\.[^\.]*$|^artist_info|credit|999nhnl\.|^group\.[^\.]*$/i } @files;
```

Comment:
Yeah this is starting to feel a bit too specific for me ngl:  

- Aren't the 9999* ones already ending up at the end by default?  
- I'd rather avoid translator-specific lingo since that gets obsolete fast -- `artist_info` and `notes` make sense to me however. (You could also probably keep `end_card_save_file` by just shortening it to `end`? False positives might be too much on that one, although it'd also catch `endcover` which might be convenient)
- As you mentioned it gets exponentially harder to avoid false positives with such a large regex, so I'd rather keep it lean.

Reply:
>Aren't the 9999* ones already ending up at the end by default?

Nope, pages with letters are sorted to be after numbers for instance. There's a ton of galleries where pages are e.g pg001.jpg but then a credit page could start with a number.

>I'd rather avoid translator-specific lingo since that gets obsolete fast -- artist_info and notes make sense to me however. (You could also probably keep end_card_save_file by just shortening it to end? False positives might be too much on that one, although it'd also catch endcover which might be convenient)

I get that, but think we should at least try and do it for popular TL filenames that are overly specific. E.G red lantern or NHNL. Their filenames shouldn't cause any issues with false positives, and it affects a lot of galleries.